### ISSUE TITLE
Unable to execute namebench.py in ubuntu

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeevandongre/namebench#76]
```
Ubuntu 9.10
Python 2.6.4

Traceback (most recent call last):
  File "./namebench.py", line 77, in <module>
    namebench = interface(options, supplied_ns, global_ns, regional_ns,
sanity_checks, version=VERSION)
TypeError: __init__() got multiple values for keyword argument 'version'



```

Original issue reported on code.google.com by `rse43.rs...@gmail.com` on 21 Dec 2009 at 5:45

**Comment by GoogleCodeExporter** [comment]
```
Delete "sanity_checks" on that line and everything should work fine.
```

Original comment by `acle...@gmail.com` on 25 Dec 2009 at 1:27

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in r536 - included in namebench 1.1b1.
```

Original comment by `thomas.r...@gmail.com` on 3 Jan 2010 at 11:05
- Changed state: **Fixed**
